appveyor.yml

```diff
@@ -0,0 +1,32 @@
+# version format
+version: 1.0.{build}-{branch}
+
+# Do not build on tags (GitHub and BitBucket)
+skip_tags: true
+
+# Do not build feature branch with open Pull Requests
+skip_branch_with_pr: true
```

Comment:
@jamesward  - Yes, you're [right](https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions). Thanks for clarifying.